Assets/Scripts/MovementSystem.cs

```diff
@@ -0,0 +1,69 @@
+using UnityEngine;
+
+public class MovementSystem
+{
+    private readonly float _speed;
+    private readonly float _sprintSpeed;
+    private readonly float _jumpForce;
+    private bool _inAir = true;
+    private readonly GameObject _gameObject;
+
+    private delegate void MoveVar(float h, float v);
+
+    private MoveVar _move;
+
+    public MovementSystem(GameObject gameObject, float speed, float sprintSpeed, float jumpForce)
+    {
+        _gameObject = gameObject;
+        _speed = speed;
+        _sprintSpeed = sprintSpeed;
+        _jumpForce = jumpForce;
+    }
+
+    public bool InSprint { get; set; } = false;
+
+    public bool InAir => _inAir;
+
+    public void Jump()
+    {
+        if (!InAir)
+        {
+            _gameObject.GetComponent<Rigidbody>().AddForce(Vector3.up * _jumpForce);
```

Comment:
As I know, `GetComponent` is complex operation, you should get it only once in constructor